CHANGELOG.md

```diff
@@ -7,6 +7,188 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [16.0.0] - 2023-08-XX
+
+### Breaking changes
+
+-   Now only supporting CDI 4.0
+-   Now supporting FDI 1.18
+-   removed the recipe specific `User` type, now all functions are using the new generic `User` type
+-   The `fetchValue` callback of claims now take a new `recipeUserId` param
+-   Now ignoring protected props in the payload in `createNewSession` and `createNewSessionWithoutRequestResponse`
+-   `createdNewUser` has been renamed to `createdNewRecipeUser` in sign up related APIs and functions
+
+-   EmailPassword:
+    -   removed `getUserById`, `getUserByEmail`
+    -   added `consumePasswordResetToken`
+    -   added an overrideable `createNewRecipeUser` function that is called during sign up and in the “invitation link” flow
+    -   `recipeUserId` is added to the input of `getContent` of the email delivery config
+    -   `email` was added to the input of `createResetPasswordToken` , `sendResetPasswordEmail`, `createResetPasswordLink`
+    -   `updateEmailOrPassword` :
+        -   now takes `recipeUserId` instead of `userId`
+        -   can return the new `EMAIL_CHANGE_NOT_ALLOWED_ERROR` status
+    -   `signIn`:
+        -   returns `recipeUserId`
+    -   `signUp`:
+        -   returns `recipeUserId`
```

Comment:
- these return the whole new user object now right? Not just recipeUserId.
- Also, these functions now attempt to link accounts as well

Reply:
- Yup, but they already did that, only the type change (which is mentioned elsewhere)
- They do, but only if you turn on account-linking which is a whole separate docs. I'll add a few lines about that pointing to the right docs.